src/Entrust/EntrustServiceProvider.php

```diff
@@ -50,6 +53,41 @@ public function register()
     }
 
     /**
+     * Register the blade directives
+     *
+     * @return void
+     */
+    private function bladeDirectives()
+    {
+        // Call to Entrust::hasRole
+        Blade::directive('role', function($expression) {
+            return "<?php if (\\Entrust::hasRole{$expression}) : ?>";
+        });
+
+        Blade::directive('endrole', function($expression) {
+            return "<?php endif; // Entrust::hasRole ?>";
+        });
+
+        // Call to Entrust::can
```

Comment:
I like better permission for consistency:

`@role` checks for role
`@permission` checks for permission 
`@ability` checks for ability